openpype/hosts/houdini/plugins/publish/extract_vdb_cache.py

```diff
@@ -14,8 +14,12 @@ class ExtractVDBCache(publish.Extractor):
     label = "Extract VDB Cache"
     families = ["vdbcache"]
     hosts = ["houdini"]
+    targets = ["local", "remote"]
 
     def process(self, instance):
+        if instance.data.get("farm"):
+            self.log.debug("Should be processed on farm, skipping.")
+            return
```

Comment:
If we have this `if instance.data.get('farm")` check then why should we still need to set the targets like this?

Reply:
I think it was already implemented in #3225 

https://github.com/ynput/OpenPype/blob/40037b050ce287f503bf4214f63e9e9f1b8028f5/openpype/modules/deadline/plugins/publish/collect_publishable_instances.py#L36

Reply:
I think it was inspired from Maya extract point cache. 
does setting targets to remote cause some plugins to be skipped ? 

https://github.com/ynput/OpenPype/blob/40037b050ce287f503bf4214f63e9e9f1b8028f5/openpype/hosts/maya/plugins/publish/extract_pointcache.py#L30-L32